src/containers/forms/PlanForm/tests/helpers.test.ts

```diff
@@ -0,0 +1,54 @@
+import { InterventionType } from '../../../../store/ducks/plans';
+import { PlanSchema } from '../helpers';
+
+/** Answers : among the validation errors raised, is there an error obj
+ * associated with the caseNum property resulting from caseNum being required
+ */
+const isCaseNumErrorPresent = (errors: any): boolean => {
+  const caseNumAssociatedErrors = errors.inner.filter(
+    (err: any) => err.path === 'caseNum' && err.message === 'Required'
```

Comment:
Do we have to use `any` here?

Reply:
will investigate and refactor accordingly

Reply:
Haven't found any useful leads on this yet, might need some help on this and yup schema type declaration.